lib/audit_test.pm

```diff
@@ -30,6 +30,8 @@ our @EXPORT = qw(
   run_testcase
   parse_testcase_log
   compare_run_log
+  lxj_compare_run_log
```

Comment:
Xiaojing, I suggest you use common variables here:) rather with your own name

Reply:
I haven't changed the origin functions, we could discuss which one we prefer. After that I will change the code again.